difumo_labels/labels_1024_dictionary.csv

```diff
@@ -279,7 +279,7 @@ Component,Difumo_names,Yeo_networks7,Yeo_networks17,GM,WM,CSF
 278,Cerebrospinal fluid (between pre and postcentral gyri and skull LH),SomMotA,SomMotA,0.492743466418865,0.09117036962411007,0.1711946506645467
 279,Supramarginal gyrus antero-inferior LH,SalVentAttnA,SalVentAttnA,0.6087415143062671,0.3243676653575063,0.06669666001709104
 280,Midbrain anterior,No network found,No network found,0.5391377479808771,0.3192916281780578,0.14069852353797596
-281,Thalamus,No network found,No network found,0.6287736887224448,0.37074135780223294,0.0004947847530555517
+281,Thalamus (middle),No network found,No network found,0.6287736887224448,0.37074135780223294,0.0004947847530555517
```

Comment:
I am commenting on giving names.

Thalamus middle or Thalamus (middle) or Thalamus, middle.

With respect to other names, we never wrote with brackets.

Simply I would prefer "Thalamus middle" to be consistent.
https://parietal-inria.github.io/DiFuMo/512/html/298.html

Reply:
OK, I like your suggestion. Let's see what other's say, and if there are no other comments, I will implement it.

Reply:
Of course, one could also argue that we can put commas, like in here:https://parietal-inria.github.io/DiFuMo/256/html/133.html

Reply:
I agree we shouldn't put brackets. I also don't like having the same name just with a comma, but we weren't as careful about that in the lower dimensions.